full-service/src/db/view_only_account.rs

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) 2020-2021 MobileCoin Inc.
+
+//! DB impl for the View Only Account model.
+
+use crate::db::{
+    models::{NewViewOnlyAccount, ViewOnlyAccount},
+    WalletDbError,
+};
+use diesel::{
+    prelude::*,
+    r2d2::{ConnectionManager, PooledConnection},
+    RunQueryDsl,
+};
+use mc_crypto_digestible::{Digestible, MerlinTranscript};
+use mc_crypto_keys::{RistrettoPrivate, RistrettoPublic};
+use std::{fmt, str};
+
+pub const DEFAULT_FIRST_BLOCK_INDEX: u64 = 0;
+pub const ROOT_ENTROPY_KEY_DERIVATION_VERSION: u8 = 1;
+pub const MNEMONIC_KEY_DERIVATION_VERSION: u8 = 2;
```

Comment:
Unsure about the `DEFAULT_FIRST_BLOCK_INDEX` value, but the other two look like they should live in mobilecoin/transaction/core/src/constants.rs

Reply:
I don't want to add these to the mobilecoin repo, I want to keep them in full-service but I agree they should be in a constants file somewhere. Will add that